src/content/pages/remote.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: EuroPython 2025 Remote
+subtitle: Join EuroPython 2025 remotely!
+toc: true
+---
+
+import { Image } from "astro:assets";
+import discord from "./images/discord.png";
+import remote from "./images/remote.png";
+
+# EuroPython 2025 Remote
+
+As happy as we are about finally being able to see many friends in [Prague](/prague)
+this year, we are also aware that many of you will not be able to make it in-person.
+Again, we present you EuroPython 2025 remote:
+Wherever you are based, let's celebrate Python and our community, rejoice at our diversity,
+learn and have fun in Prague and beyond! In this spirit, we strive to make the remote participation
+as interactive as possible.
+<Image src={remote} width="1200" alt="" />
+
+## How to participate in the conference remotely?
+
+- To participate in the conference interactively (and have guaranteed access to livestreams), a "remote ticket" registration is required. See details and pricing on the [Tickets information](/tickets#remote-tickets) page.
+- All ticket holders will receive a link to join our [Discord server](/discord) for the conference.
```

Comment:
Let's remove this as well, just so we have less random people joining the server without the tickets :sweat_smile: